build.gradle

```diff
@@ -38,22 +38,22 @@ ext {
     versionName = version
 }
 
-ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID') ?: ''
-ext["signing.password"] = System.getenv('SIGNING_KEY_PASSWORD') ?: ''
-ext["signing.secretKeyRingFile"] = System.getenv('SIGNING_KEY_FILE') ?: ''
-
 allprojects {
     repositories {
         mavenCentral()
         google()
+
+        maven {
+            url 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
```

Comment:
q: is this mostly to help with testing snapshot artifacts?

Reply:
Actually this was just copied over from the paypal-android gradle file but I don't think it's necessary here - removed.